go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/livekit/livekit-server
 
-go 1.17
+go 1.18
```

Comment:
Are we okay with this? I have some vague recollection of something failing with this. Maybe, @cnderrauber faced it? Or maybe it was from community.

Reply:
We'd lose a generated CLI flag for a config field that's a pointer (`rtc.allow_tcp_fallback`), but maybe that's OK.

Reply:
Correction - `reflect.Ptr` works for backwards compatibility 👍